lib/parser.js

```diff
@@ -323,6 +323,44 @@ function handleShh(keys)
   return statement;
 }
 
+/**
+ * Handles the beginning of a multi-line comment:
+ * quiet [comments]
+ */
+ function handleQuiet(keys)
+ {
+   expectToken('quiet', keys);
+
+   var statement = '/*';
+   multiComment = true;
+   for (var i = 1; i < keys.length; i++) {
+       statement += ' ' + keys[i];
+   }
+   statement += '\n';
+
+   return statement;
+ }
+
+/**
+ * Handles the end of the multi-line comment:
+ * loud
+ */
+function handleLoud(keys)
+{
+  expectToken('loud', keys);
+  if(!multiComment)
+  {
+    throw new Error("Unparseable syntax! Encountered: 'loud' without first seeing 'quiet'");
+  }
+
+  var statement = '*/';
+  multiComment = false;
+  for (var i = 1; i < keys.length; i++) {
```

Comment:
not entirely sure why we had this in the first place since it seems it just processes the rest as text:
```
quiet
a
loud very foo is bar
```

Yields
```
/*
a
*/ very foo is bar
```

Once everything is extracted out, we can change things to shift the keys (consume the tokens) and potentially be able to have `quiet a loud` -> `/* a */`